ACE/ace/config-sunos5.10.h

```diff
@@ -59,4 +59,9 @@
 
 #define ACE_HAS_SOLARIS_ATOMIC_LIB
 
+// Solaris Studio 12.4 implements symbol lookup correctly.
+#if defined (__SUNPRO_CC) && (__SUNPRO_CC >= 0x5130)
+#define ACE_ANY_OPS_USE_NAMESPACE
+#endif
```

Comment:
Wouldn't it be better to include config-suncc-common.h in the base config-sunos file at the moment suncc is used? That way we only have this knowledge once